apps/alchimie/alchimie/assets.py

```
@@ -1 +1,18 @@
+import dagster as dg
 
+
+@dg.asset
+def my_asset():
+    return "Hello, world!"
+
+
+@dg.asset(required_resource_keys={"postgres"})
+def postgres_query_asset(context):
```

Ruff (0.8.2)

Missing return type annotation for public function `postgres_query_asset`

(ANN201)